packages/api/src/tasks/checkAlerts.ts

```diff
@@ -100,10 +98,11 @@ export const buildChartLink = ({
 };
 
 export const doesExceedThreshold = (
-  isThresholdTypeAbove: boolean,
+  thresholdType: AlertThresholdType,
```

Comment:
Will the alert crash if customer set up an alert with unknown type?

Reply:
Theoretically, users shouldn't be able to set the arbitrary type since the endpoint validator enforces it (https://github.com/hyperdxio/hyperdx/blob/679092981271e33e1b536a3484f290c31d6710df/packages/api/src/routers/api/alerts.ts#L93)